### PR TITLE
Fix the mistake about Neo4j Helm chart and service accounts

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1590.surge.sh

Updated pages:
- [Authentication and authorization](https://neo4j-docs-operations-1590.surge.sh/operations-manual/5/kubernetes/authentication-authorization)

File: modules/ROOT/pages/kubernetes/authentication-authorization.adoc [hunk 1]
```diff
@@ -89,5 +89,5 @@ podSpec:
 [NOTE]
 ====
 The service account must already exist.
-The Neo4j Helm chart does not create or configure Service Accounts.
+In case of a cluster the Neo4j Helm chart creates or configures service accounts.
```
Comment (renetapopova) [inline modules/ROOT/pages/kubernetes/authentication-authorization.adoc:92]: I think it should be like this, but let Harshit confirms it.
```suggestion
In the case of clusters, the Neo4j Helm chart creates the service account.
```

**Comment by renetapopova** [comment]
This change can't be cherry-picked into 4.4 directly but copied as 4.4 is structured differently.

**Review by harshitsinghvi22 — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1590.surge.sh

Updated pages:
- [Authentication and authorization](https://neo4j-docs-operations-1590.surge.sh/operations-manual/5/kubernetes/authentication-authorization)

**Pull request merged**